forwarder/vppagent/pkg/converter/utils.go

```diff
@@ -1,12 +1,20 @@
 package converter
 
 import (
+	"fmt"
 	"net"
+	"os"
 
 	"github.com/rs/xid"
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	ForwarderAllowVHost = "FORWARDER_ALLOW_VHOST" // To disallow VHOST please pass "false" into this env variable.
```

Comment:
DO we want this in `docs/env.md`?

Reply:
I moved this const from `forwarder/vppagent/pkg/converter/kernel_connection_converter.go` to `utils.go`. 
Should I add it to docs/env?

Reply:
Yes please, we'd betetr have it documented in a way.